test/modules/trackpad-scroll.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Unit tests for trackpad scroll speed normalization
+ */
+
+QUnit.module('Trackpad Scroll Speed', function() {
+    
+    QUnit.test('MouseTracker trackpadScrollSensitivity property', function(assert) {
+        // Test default value
+        const tracker1 = new OpenSeadragon.MouseTracker({
+            element: document.createElement('div')
+        });
+        assert.equal(tracker1.trackpadScrollSensitivity, 0.3, 'Default trackpadScrollSensitivity should be 0.3');
+        
+        // Test custom value
+        const tracker2 = new OpenSeadragon.MouseTracker({
+            element: document.createElement('div'),
+            trackpadScrollSensitivity: 0.5
+        });
+        assert.equal(tracker2.trackpadScrollSensitivity, 0.5, 'Custom trackpadScrollSensitivity should be set correctly');
+    });
+
+    QUnit.test('Trackpad detection logic', function(assert) {
+        const tracker = new OpenSeadragon.MouseTracker({
+            element: document.createElement('div'),
+            trackpadScrollSensitivity: 0.3
+        });
+
+        // Mock wheel events
+        const trackpadEvent = {
+            deltaY: 5,
+            deltaMode: 0, // pixel mode
+            shiftKey: false,
+            clientX: 100,
+            clientY: 100
+        };
+
+        const mouseWheelEvent = {
+            deltaY: 100,
+            deltaMode: 1, // line mode
+            shiftKey: false,
+            clientX: 100,
+            clientY: 100
+        };
+
+        // Test trackpad detection
+        const isTrackpad1 = Math.abs(trackpadEvent.deltaY) < 10 && trackpadEvent.deltaMode === 0;
```

Comment:
This doesn't feel like it's actually testing the MouseTracker... it seems it would be better to be testing the mousepad version of this logic rather than duplicating the logic here.